pkg/sosreport/run-sosreport.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+if grep -q 'CPE_NAME=.*atomic-host' /etc/os-release; then
```

Comment:
Good point about the container. No way to predict that. However the CPE_NAME thingy above doesn't help with using the right image.